### PR TITLE
/open pr: handle OpenSSL correctly

### PR DESCRIPTION
**Pull request by dscho**
This teaches Git for Windows' automation to avoid meddling with the OpenSSL packages' `pkgver` lines; They are designed to transmogrify the official OpenSSL version into a string that is acceptable as a version string to Pacman.

### CODE REVIEW

File: update-scripts/version/mingw-w64-openssl [hunk 1]
```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env node
+
+// Handle this identically to the MSYS package
+require('./openssl')
```
Comment (rimrul) [inline update-scripts/version/mingw-w64-openssl:4]: good

File: update-scripts/version/openssl [hunk 2]
```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env node
+
+// The `openssl` version is not actually stored in the `pkgver` variable, but in
+// `_ver` (because of the funny trailing letter in the OpenSSL v1.1.* versions)
+
+(async () => {
+    const version = process.argv[2].replace(/_P/, 'p')
```
Comment (rimrul) [inline update-scripts/version/openssl:7]: Thos seems to be a leftover from OpenSSH.

**Review by rimrul — CHANGES_REQUESTED** [review]
There's a small leftover from SSH, but the rest looks good.

File: update-scripts/version/openssl (hunk 2 above)
Comment (dscho) [reply update-scripts/version/openssl:7]: It totally is! Thank you for that.

Comment (dscho) [reply update-scripts/version/openssl]: @rimrul this is now [fixed](https://github.com/git-for-windows/git-for-windows-automation/compare/6e5b5499098331e04fa87e1279a38cc04a956f9a..cb2053fe974cbce4ea00f640ab46d10376e7e0cb)!

Comment (rimrul) [reply update-scripts/version/openssl]: You where a bit overzealous. I meant just the `.replace()` you got rid of the whole line. We still need the `version` variable later, to write it to the `_ver` line

Comment (dscho) [reply update-scripts/version/openssl]: Oh my. 🤦

Comment (dscho) [reply update-scripts/version/openssl]: [Fixed again](https://github.com/git-for-windows/git-for-windows-automation/compare/cb2053fe974cbce4ea00f640ab46d10376e7e0cb..f7c15cb1a7d28fabeeae4060d867f65611c07135).

**Review by rimrul — APPROVED** [review]

**Pull request merged**